### ISSUE TITLE
default-features = false seems to be broken on 0.8

### ISSUE THREAD

**Issue by Ralith** [quinn-rs/quinn#1251]
This Cargo.toml:
```toml
[package]
name = "test-quinn"
version = "0.1.0"
edition = "2021"
resolver = "2"

[dependencies]
quinn = { version = "0.8", default-features = false }
```
is still pulling in rustls, ring, webpki, etc. somehow. quinn's dependency on quinn-proto does specify `default-features = false` correctly, and quinn doesn't pull in its own optional webpki dep, so I'm at a loss.

**Comment by daxpedda** [comment]
I believe it's because quinn also pulls in quinn-udp, which in turn is pulling in quinn-proto too, but is missing `default-features = false`.

**Comment by Ralith** [comment]
Great catch, thanks!

**Comment by Ralith** [comment]
Fixed in #1252.
